tests/plugins/test_koji.py

```diff
@@ -172,11 +183,11 @@ def test_koji_plugin(self,
             'target': target,
             'hub': '',
             'root': root,
-            'proxy': proxy
+            'proxy': proxy,
         }
 
         if koji_ssl_certs:
-            args['koji_ssl_certs'] = str(tmpdir)
+            args['koji_ssl_certs_dir'] = str(tmpdir)
             with open('{}/ca'.format(tmpdir), 'w') as ca_fd:
                 ca_fd.write('ca')
```

Comment:
Is this file ever read during the test? It isn't created for tests for the other plugin.